### PR TITLE
added message when user logged into their account

### PR DESCRIPTION
**Pull request by zlrkw11**
image

### CODE REVIEW

File: client/src/components/composite/LoginForm/LoginForm.tsx [hunk 1]
```diff
@@ -57,6 +57,7 @@ const LoginForm = ({
     if (isLoading || !loginHandler) return
     try {
       setIsLoading(true)
+      alert("Logging In...")
```
Comment (bcho892) [inline client/src/components/composite/LoginForm/LoginForm.tsx:60]: we want something to display under the button, not an alert (which blocks everyting)

**Review by bcho892 — COMMENTED** [review]
The PR says

**Comment by zlrkw11** [comment]
Easy, I wanted to make a component but thought alert was needed

**Comment by zlrkw11** [comment]
https://github.com/user-attachments/assets/1794f6c8-6661-4b92-bcc5-324bceecebcb

**Comment by zlrkw11** [comment]
https://github.com/user-attachments/assets/7b7e6d19-a4e6-4789-a22a-62d171360ecd

updated

File: client/src/components/composite/LoginForm/LoginForm.tsx [hunk 2]
```diff
@@ -134,10 +134,14 @@ const LoginForm = ({
...
             Login
           </Button>
+
           <Link href="/register" className="flex">
             <Button variant="inverted-default-sm">Register</Button>
           </Link>
         </span>
+        {isLoading ? (
+          <p className="text-dark-blue-100 absolute mt-4">Logging in...</p>
+        ) : null}
```
Comment (bcho892) [inline client/src/components/composite/LoginForm/LoginForm.tsx:144]:
```suggestion
        {isLoading && (
          <p className="text-dark-blue-100 absolute mt-4">Logging in...</p>
        )}
```

**Review by bcho892 — APPROVED** [review]

**Review by bcho892 — CHANGES_REQUESTED** [review]
shouldn't be absolute, causes styling problems

**Review by bcho892 — APPROVED** [review]

**Pull request merged**